website/content/docs/troubleshoot/common-errors.mdx

```diff
@@ -147,6 +149,40 @@ You have installed an Enterprise version of Consul. If you are an Enterprise cus
 
 -> **Note:** Enterprise binaries can be identified on our [download site][releases] by the `+ent` suffix.
 
+## Common errors on Kubernetes
+
+### Unable to connect to the Consul client on the same host
+
+```text
+Put http://10.20.62.49:8500/v1/catalog/register: dial tcp 10.20.62.49:8500: connect: connection refused
+```
+
+```text
+Put http://172.16.30.51:8500/v1/agent/service/register: dial tcp 172.16.30.51:8500: connect: connection refused
+```
+
+```text
+Get http://10.137.120.213:8500/v1/status/leader: dial tcp 10.137.120.213:8500: i/o timeout
+```
+
+If you're seeing `i/o timeout` or `connection refused` errors when connecting to the Consul client on the Kubernetes worker,
```

Comment:
Non blocking suggestion - move some of this paragraph before the commands to provide context for what you are about to show.  Something like

### `i/o timeout` or `connection refused` errors 

If you're seeing `i/o timeout` or `connection refused` errors when connecting to the Consul client on the Kubernetes worker, this could be because the CNI (Container Networking Interface) does not [support](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#support-hostport)
the use of `hostPort`. 

Then show the commands.  Then explain the work around.

Reply:
will do!